validator/remote/src/main/java/tech/pegasys/teku/validator/remote/BeaconNodeReadinessManager.java

```diff
@@ -201,4 +209,24 @@ private void processNotReadyResult(
       beaconNodeReadinessChannel.onFailoverNodeNotReady(beaconNodeApi);
     }
   }
+
+  public enum ReadinessStatus {
+    NOT_READY(0),
+    OPTIMISTIC(1),
+    READY(2);
+
+    private final int weight;
+
+    ReadinessStatus(final int weight) {
+      this.weight = weight;
+    }
+
+    int getWeight() {
```

Comment:
No need of getWeight actually since can directly access `.weight` on inner classes

Reply:
Removed it